packager/mpd/base/xml/xml_node_unittest.cc

```diff
@@ -543,5 +546,185 @@ TEST_F(LiveSegmentTimelineTest, LastSegmentNumberSupplementalProperty) {
   FLAGS_dash_add_last_segment_number_when_needed = false;
 }
 
+class OnDemandVODSegmentTest : public ::testing::Test {
+ protected:
+  void SetUp() override {}
+
+  void TearDown() override {}
+
+  MediaInfo ConvertToMediaInfo(const std::string& media_info_string) {
+    MediaInfo media_info;
+    CHECK(::google::protobuf::TextFormat::ParseFromString(media_info_string,
+                                                          &media_info));
+    return media_info;
+  }
```

Comment:
Can you include mpd/test/mpd_builder_test_helper.h instead of redefining `ConvertToMediaInfo`?